packages/typescript/src/internal/build_defs.bzl

```diff
@@ -269,6 +270,15 @@ def _ts_library_impl(ctx):
         devmode_compile_action = _devmode_compile_action,
         tsc_wrapped_tsconfig = tsc_wrapped_tsconfig,
     )
+
+    # Add in new JS providers
```

Comment:
link to issue or design about adding more and deleting the "typescript" one

Reply:
👍